sci-misc/boinc/boinc-8.2.1.ebuild

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV=$(ver_cut 1-2)
+WX_GTK_VER=3.2-gtk3
+
+inherit autotools bash-completion-r1 linux-info optfeature wxwidgets xdg-utils
+
+DESCRIPTION="The Berkeley Open Infrastructure for Network Computing"
+HOMEPAGE="https://boinc.berkeley.edu/"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/BOINC/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/BOINC/boinc/archive/client_release/${MY_PV}/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+	S="${WORKDIR}/${PN}-client_release-${MY_PV}-${PV}"
+fi
+
+LICENSE="LGPL-3+"
+SLOT="alpha"
+
+# - "X" builds idle tracking via XScrnSaver and the libboinc_graphics2 library
+# - "gui" builds the manager app and the screensaver
+IUSE="X gui"
+
+DEPEND="
+	>=acct-user/boinc-1
+	app-misc/ca-certificates
+	dev-libs/openssl:=
+	net-misc/curl[ssl]
+	sys-libs/zlib:=
+	X? (
+		media-libs/freeglut
+		media-libs/libjpeg-turbo:=
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+	)
+	gui? (
+		dev-libs/glib:2
+		x11-libs/gtk+:3
+		x11-libs/libnotify
+		x11-libs/libxcb:=
+		x11-libs/wxGTK:${WX_GTK_VER}=[opengl,webkit]
+		x11-libs/xcb-util
+	)
+"
+RDEPEND="
+	${DEPEND}
+	!sci-misc/boinc:0
```

Comment:
Please don't use slots if they're not parallel installable. Use subslots instead.

Reply:
What kind of subslots? "0/stable" and "0/alpha"?

This package also installs some libraries and header files, but they are not enough to build an application against them. Maybe it's worth removing them altogether.